debug_toolbar/toolbar.py

```diff
@@ -88,22 +88,8 @@ def should_render_panels(self):
             render_panels = self.request.META["wsgi.multiprocess"]
         return render_panels
 
-    # Handle storing toolbars in memory and fetching them later on
-
-    _store = OrderedDict()
-
     def store(self):
-        # Store already exists.
-        if self.store_id:
-            return
-        self.store_id = uuid.uuid4().hex
-        self._store[self.store_id] = self
-        for _ in range(self.config["RESULTS_CACHE_SIZE"], len(self._store)):
-            self._store.popitem(last=False)
-
-    @classmethod
-    def fetch(cls, store_id):
-        return cls._store.get(store_id)
+        get_store().set(self.store_id)
```

Comment:
now i would:

`return import_string(dt_settings.get_config()["TOOLBAR_STORE_CLASS"])`

are these edits/changes functional and complete?

Reply:
The function `store` here is more the verb case of "store". What's happening is that we want to store this version of the toolbar into the storage mechanism. The naming could be improved.